### ISSUE TITLE
[kube-apiserver] Auditing failed of request: encoding failed no kind "Scale" is registered for version "apps/v1"

### ISSUE THREAD

**Issue by alexfanat86** [kubernetes/kubernetes#109720]
### What happened?

Hello! After upgrading our k8s-cluster to v1.21.11 we start to see a lot of warnings in kube-apiserver logs:
`Auditing failed of  request: encoding failed: no kind "Scale" is registered for version "apps/v1" in scheme "k8s.io/kubernetes/pkg/api/legacyscheme/scheme.go:30"`
Probably these warnings can lead to increased apiserver latency.

### What did you expect to happen?

We expect not to see this warnings in kube-apiserver logs

### How can we reproduce it (as minimally and precisely as possible)?

Not sure how to reproduce this but we see this watnings in all our k8s-clusters after upgrading to v1.21.11

### Anything else we need to know?

We tried to avoid auditing of this by adding section to our `/etc/kubernetes/audit/audit-policy.yaml`
```
  # Don't audit Scale events
  - level: None
    resources:
      - group: "apps"
        resources: ["deployments/scale", "replicasets/scale", "statefulsets/scale"]
      - group: "autoscaling"
        resources: ["scale"]
```
But it doesn't work

### Kubernetes version

<details>

```console
$ kubectl version
Client Version: version.Info{Major:"1", Minor:"20", GitVersion:"v1.20.2", GitCommit:"faecb196815e248d3ecfb03c680a4507229c2a56", GitTreeState:"clean", BuildDate:"2021-01-14T05:15:04Z", GoVersion:"go1.15.6", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"21", GitVersion:"v1.21.11", GitCommit:"38d3c1f3d5306401bcf39a71bad3b5a5106033d7", GitTreeState:"clean", BuildDate:"2022-03-16T14:02:06Z", GoVersion:"go1.16.15", Compiler:"gc", Platform:"linux/amd64"}
```

</details>

### Cloud provider

<details>

</details>

### OS version

<details>

```console
# On Linux:
$ cat /etc/os-release
NAME="Ubuntu"
VERSION="18.04.5 LTS (Bionic Beaver)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 18.04.5 LTS"
VERSION_ID="18.04"
$ uname -r
5.10.84-051084-generic
```

</details>

### Install tools

<details>

</details>

### Container runtime (CRI) and version (if applicable)

<details>

</details>

### Related plugins (CNI, CSI, ...) and versions (if applicable)

<details>

</details>

**Comment by vsxen** [comment]
try to upgrade your kubectl client ?
 `Client Version: version.Info{Major:"1", Minor:"20", GitVersion:"v1.20.2"`

**Comment by alexfanat86** [comment]
Hello! I upgraded my `kubectl` client, but in won't help. There are still this warning messages. I'm not the only one who have access to k8s-clusters. But system administrators doesn't try to get Scale resources and developers doesn't have any permissions for Scale resources. I still don't realize why this warning messages appear.

**Comment by vsxen** [comment]
https://github.com/kedacore/keda/issues/1646

**Comment by alexfanat86** [comment]
> https://github.com/kedacore/keda/issues/1646

I saw this issue but we do not have any keda operator in our clusters

**Comment by liggitt** [comment]
I think this is the same as https://github.com/kubernetes/kubernetes/issues/98612, which was resolved in 1.22 in https://github.com/kubernetes/kubernetes/pull/102040
